### PR TITLE
Add basic CDC chart with data from GraphQL API

### PR DESCRIPTION
**Pull request by jeremiak**
Still needs some styling work but good enough for review/merge and we can break out other issues to fix the small things.

Screen Shot 2020-04-09 at 3 48 28 PM

### CODE REVIEW

File: src/pages/dashboard/_CDCComparisonContainer.js [hunk 1]
```diff
@@ -1,64 +1,101 @@
...
-          fill="#585BC1"
-          height={400}
-          marginBottom={40}
-          marginLeft={80}
-          marginRight={10}
-          marginTop={10}
-          xTicks={2}
-          width={200}
-        />
+      <div style={{ display: 'flex', flexGrow: 1, width: '50%' }}>
```
Comment (jeremiak) [inline src/pages/dashboard/_CDCComparisonContainer.js:71]: All this inline style business is going to have to change, but just wanted to get this up quickly

File: src/pages/dashboard/_CDCComparisonContainer.js [hunk 2]
```diff
@@ -1,64 +1,101 @@
...
+  const data = query.allCovidUsDaily.nodes
+    .map(node => [
+      {
+        date: parseDate(node.date),
+        value: node.totalTestResultsIncrease,
+      },
+    ])
+    .flat()
+    .sort(sortChronologically)
+  const cdcCumulativeTotal = sum(cdcData, d => d.value)
```
Comment (jeremiak) [inline src/pages/dashboard/_CDCComparisonContainer.js:54]: My understanding of the CDC data is that its all daily and none of it is cumulative, so we should be able to create a sum like this

File: src/pages/dashboard/_CDCComparisonContainer.js [hunk 3]
```diff
@@ -1,64 +1,99 @@
...
-    return nodes.sort((a, b) => {
-      if (a.date > b.date) return 1
-      if (a.date < b.date) return -1
-      return 0
+  const cdcData = query.allCdcDaily.nodes
+    .map(node => {
+      const date = parseCdcDate(`${node.dateCollected}/2020`)
+      return {
+        date,
+        value: +node.dailyTotal,
```
Comment (goleary) [inline src/pages/dashboard/_CDCComparisonContainer.js:41]: what does the plus here denote?

Comment (jeremiak) [reply src/pages/dashboard/_CDCComparisonContainer.js:41]: It coerces the value to an integer

**Pull request merged**